Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -1143,11 +1153,58 @@ void HackStudioWidget::create_project_tab(GUI::Widget& parent)
     };
 }
 
+Vector<String> HackStudioWidget::read_recent_projects() const
```

Comment:
This can be a static function.

Reply:
Changed it to a public & static method in HackStudioWidget because I ended up needing it to implement the "reopen the last project" thing. Not sure if it's the right class where to put it though